apps/sms/views/shared/js/startup.js

```diff
@@ -17,6 +17,7 @@
          Navigation,
```

Comment:
Hmm, I see we don't initialize ```ActivityHandler``` neither in split Inbox, nor in split Conversation - likely we'll have separate bug to enable ```ActivityHandler``` and ```SystemMessageHandler``` in those views once we fix https://bugzilla.mozilla.org/show_bug.cgi?id=818000